### DISCUSSION TITLE
Contributing Dutch benchmarks from the GPT-NL project

### DISCUSSION THREAD

**Discussion by Rijgersberg** [EuroEval/EuroEval#499] [General]
First of all: thank you for the great ScandEval project!

I’m a machine learning engineer at the [Netherlands Forensic Institute](https://www.forensicinstitute.nl/) (NFI). I’ve had an interest in LLMs for a while now, and you may have seen my hobby project [GEITje-7B](https://github.com/Rijgersberg/GEITje), a Dutch LLM.

The NFI is one of the partners in the [GPT-NL](https://gpt-nl.nl/) project, so I have a role in the project too. I argued that alongside the development of the model it is very important that we have good native Dutch benchmarks in order to evaluate and compare any models we create. So, long story short, I’m now the lead of the newly created “Evaluation & Benchmarking Work Package”, which we will start working on soon.

Part of my proposal was to
1.	Do as much as possible of the work in public
2.	Use and contribute to open source projects instead of reinventing the wheel.

So my idea was to contribute any benchmarks we make to ScandEval.

My question to you: can you take a look at the brief description of our plans below, and give some feedback on which of the three types of benchmarks you think would be a good fit for ScandEval?

Thanks!

> **Description of work**
> -	Make a selection of the highest priority types of benchmarks that match the most current understanding of GPT-NL use cases. We want to do that openly with input from academia and the Dutch open LLM community
>   1.	Task performance in Dutch (reasoning, instruction following, summarization and simplification)
>   2.	Dutch cultural understanding and linguistic abilities
>   3.	Bias and fairness in Dutch
> -	Create the highest priority benchmarks from each of those three categories by:
>   -	Acquiring raw native Dutch datasets (no automatic translations!)
>   -	Filtering, cleaning and processing them into benchmark-ready datasets
>   -	Adding them to an open source evaluation package
> -	Repeat until the allotted time budget has run out or the gains are no longer worth the effort
> -	Run each GPT-NL model on the benchmarks and compare with existing (Dutch) models
> -	Open source all the code and datasets
> 
> Optionally: contribute to an academic publication about the benchmarks.

**Comment by saattrupdan** [comment]
Hi @Rijgersberg! And of course I know of your great work with the GEITje models 🇳🇱 

Fantastic to hear that efforts are put into creating more gold-standard evaluation datasets - we need them!

Here are some comments related to your three categories:

> Task performance in Dutch (reasoning, instruction following, summarization and simplification)

One task that is currently needed in Dutch is an extractive question answering dataset. I.e., a dataset with (context, question, answer) triples, akin to SQuAD. Currently in ScandEval we're working with a machine translated version of the English SQuAD dataset, which is not ideal. It is also the only language in ScandEval that has a machine translated QA dataset.

Common-sense reasoning is also machine translated for Dutch currently, based on the American HellaSwag dataset, so a gold standard alternative to this would also be preferred. Aside from being machine translated, it has [also been found to have many errors](https://www.surgehq.ai/blog/hellaswag-or-hellabad-36-of-this-popular-llm-benchmark-contains-errors), making it even more important to replace it with a more trustworthy dataset.

> Dutch cultural understanding and linguistic abilities

This would be highly useful, and would fit directly in the "knowledge" category in the benchmark. In Danish we have done the same thing, using past Danish citizenship tests for measuring cultural understanding and meanings of Danish idioms for testing knowledge of Danish sayings (which cannot be captured simply by translating). It seems like you have a similar kind of citizenship test resource in the Netherlands, and I've posted [an issue](https://github.com/ScandEval/ScandEval/issues/391) with some links that could be useful to you here.

> Bias and fairness in Dutch

We're actually currently working with TNO on this (alongside other languages as well). We're still in the beginning stages, so you are more than welcome to join. But for this reason I'd also say that the other categories are more lacking at the moment.

**Reply by Rijgersberg** [reply]
Thanks, that great input! I'll contact your collaborators inside TNO for the bias and fearness subject.

You will see more of use when we get started :)
